doc/nrf/includes/matter_sample_wifi_flash.txt

```diff
@@ -0,0 +1,27 @@
+In the Matter over Wi-Fi variant of this sample, the :ref:`a portion of the application code related to the nRF70 Series' Wi-Fi firmware <ug_nrf70_developing_fw_patch_ext_flash>` is stored in external memory.
+When you flash the sample onto your development kit, the build process automatically includes the Wi-Fi firmware patch and programs it into the external memory.
+
+By default, using the ``west flash`` command will erase the entire external flash memory before writing the new Wi-Fi firmware patch.
+This full-chip erase ensures that the patch is written to a clean memory area, but it can significantly increase flashing time—sometimes taking several minutes, especially if the external flash is large.
```

Comment:
```suggestion
This full-chip erase ensures that the patch is written to a clean memory area, but it can significantly increase flashing time - sometimes taking several minutes, especially if the external flash is large.
```